src/solver/conv_winoRxS_fused.cpp

```diff
@@ -417,7 +409,12 @@ ConvSolution ConvBinWinogradRxSf2x3g1Fused::GetSolution(const FusionContext& con
     };
     return result;
 }
-
+float ConvBinWinogradRxSf2x3g1Fused::GetWti(const FusionContext& ctx,
+                                            const FusionDescription& problem) const
+{
+    const auto conv_problem = problem.GetConvProblem(0, miopen::conv::Direction::Forward);
+    return conv::ConvBinWinogradRxSf2x3g1().GetWti(ctx, conv_problem);
```

Comment:
[notice] Maybe multiply with `1.001f` just to notice some small overhead?

Reply:
I don't think it would be useful because WTI is not meant to be comparable between primitives. I think multiplication by 1.001 would mean that the rest of the kernel is .1% less optimal than non-conv part of its competitors.